backend/package.json

```diff
@@ -17,7 +17,8 @@
     "helmet": "^4.6.0",
     "http-status-codes": "^2.1.4",
     "joi": "^17.4.2",
-    "mongoose": "^6.0.6"
+    "mongoose": "^6.0.6",
+    "mongoose-unique-validator": "^3.0.0"
```

Comment:
remove package

Reply:
After consultation, left as additional layer of validation